src/Umbraco.Community.Contentment/DataEditors/DataPicker/IDataPickerSource.cs

```diff
@@ -4,13 +4,14 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 using System.Collections.Generic;
+using System.Threading.Tasks;
 
 namespace Umbraco.Community.Contentment.DataEditors
 {
     public interface IDataPickerSource : IContentmentEditorItem
     {
-        IEnumerable<DataPickerItem> GetItems(Dictionary<string, object> config, IEnumerable<string> values);
+        Task<IEnumerable<DataPickerItem>> GetItemsAsync(Dictionary<string, object> config, IEnumerable<string> values);
```

Comment:
I'm wondering if these really need the `Async` suffix?
If we had both sync/async methods, then I could see the distinction, but not sure if we need it here. 🤔

Reply:
It's a standard convention in C#. Helps people not forget to use the await keyword,

Reply:
Yeah, it's a Microsoft convention. They say it in their naming guidelines:

"The name of an async method, by convention, ends with an "Async" suffix." - https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/async/task-asynchronous-programming-model

"Add "Async" as the suffix of every async method name you write." - https://learn.microsoft.com/en-us/dotnet/csharp/async

Reply:
I can totally see why to use the suffix for public consumer API methods, but _(for whatever reason)_ it's niggling me that these 2 methods are only going to be called by Contentment's WebAPI controller - they'll effectively be an internal call. Hmmm, just not sitting right with me. 😕

Reply:
The interface is public and is likely to be called by test code.

Reply:
@nzdev Touché! 😅